app/Console/Commands/ConvertToSpeech.php

```diff
@@ -57,29 +59,65 @@ public function fire()
                     $item->status = Items::STATUS_BEING_CONVERTED;
                     $item->save();
 
-                    printf("Converting content of '%s'...", $item->url);
+                    $this->info(sprintf("Converting content of '%s'...", $item->url));
+
+                    try {
+                        $response = $this->getHttpClient()->post("v1/synthesize", array(
+                            "headers" => array(
+                                'Authorization' => 'Basic ' . base64_encode(env('WATSON_TTS_USERNAME') . ':' . env('WATSON_TTS_PASSWORD')),
```

Comment:
Probably better to use the `auth` option and let Guzzle encode the authentication header for you (http://guzzle.readthedocs.org/en/latest/request-options.html#auth)